src/engine/SCons/Tool/install.xml

```diff
@@ -46,12 +46,48 @@ which must be a directory.
 The names of the specified source files or directories
 remain the same within the destination directory. The
 sources may be given as a string or as a node returned by
-a builder.
+a builder. &Install; is accumulative, additional calls
+add additional files to the list to be copied to the target.
 </para>
 
 <example_commands>
-env.Install('/usr/local/bin', source = ['foo', 'bar'])
+env.Install(target='/usr/local/bin', source=['foo', 'bar'])
 </example_commands>
+
+<para>
+Note that because of the kinds of paths often chosen for the
+&Install; builder (and the related &InstallAs; and
+&InstallVersionedLib; builders), the results can be surprising.
+The above example will not, unless arrangements are made,
+actually copy anything.
+The target is "built" by copying the source files to it,
+but the default behavior of SCons is to select for building
+only those targets that are underneath the current directory.
```

Comment:
Not really accurate.
The default behavior is to only build files under the specified target. If no target is specified then the default target is the directory scons was invoked in (Barring other command line flags which can affect such)..
I'm not sure expanding upon this here is the best, perhap a reference/link to Default(), and expanding any explanation there?

Reply:
Odd... so you're saying if the SConscript contains  `env.Program('bar', 'bar.c.') scons doesn't consider `bar` as a target, but instead considers the directory the target?

Reply:
specified target = specific command line target as in ```scons <TARGET>```, not as in the per builder target.

Reply:
terminology overloading again?

Reply:
indeed..